android/app/src/main/java/updated/mysterium/vpn/common/languages/LanguagesUtil.kt

```diff
@@ -14,7 +14,7 @@ object LanguagesUtil {
         "ZH"
     )
 
-    fun getUserDefaultLanguage() = Locale.getDefault().language.toUpperCase(Locale.ROOT)
+    fun getUserDefaultLanguage() = Locale.getDefault().language.uppercase(Locale.ROOT)
```

Comment:
With new version of `.upercase()` extension, there is no need to pass the `Locale.ROOT` parameter because this is the default parameter of a function. So that expression could be just `Locale.getDefault().language.uppercase()`
This isn’t a change request, it’s totally up to you. Leaving it as-is is absolutely fine.